src/common/services/encoding.js

```diff
@@ -0,0 +1,18 @@
+const abi = require('ethereumjs-abi');
```

Comment:
Can we use web3 for this instead or is `ethereumjs-abi` doing something we miss from web3?

https://web3js.readthedocs.io/en/v1.2.6/web3-eth-abi.html#encodefunctionsignature